lib/sidekiq/cron/job.rb

```diff
@@ -411,7 +410,7 @@ def to_hash
           active_job: @active_job,
           queue_name_prefix: @active_job_queue_name_prefix,
           queue_name_delimiter: @active_job_queue_name_delimiter,
-          last_enqueue_time: @last_enqueue_time,
+          last_enqueue_time: @last_enqueue_time.to_s,
```

Comment:
@serprex maybe this change closes #354 too?

Reply:
I don't believe so, previous redis lib would stringify parameters, while new one choked. But with timestamp it should use iso8601 to stringify. I think the other PR should be merged